webpay/settings/sites/awsdev/settings_base.py

```diff
@@ -0,0 +1,126 @@
+"""private_base will be populated from puppet and placed in this directory"""
+import private_base as private
+
+from webpay.settings import base
+
+from .. import splitstrip
+
+DOMAIN = 'marketplace.dev.mozaws.net'
+ALLOWED_HOSTS = [DOMAIN]
+
+CACHE_PREFIX = private.CACHE_PREFIX
+
+CACHES = {
+    'default': {
+        'BACKEND': 'django.core.cache.backends.memcached.MemcachedCache',
+        'LOCATION': splitstrip(private.CACHES_DEFAULT_LOCATION),
+        'TIMEOUT': 500,
+        'KEY_PREFIX': CACHE_PREFIX,
+    },
+}
+
+# Recipients of traceback emails and other notifications.
+ADMINS = ()
+MANAGERS = ADMINS
+
+# Debugging displays nice error messages, but leaks memory. Set this to False
+# on all server instances and True only for development.
+DEBUG = TEMPLATE_DEBUG = False
+
+# Is this a development instance? Set this to True on development/master
+# instances and False on stage/prod.
+DEV = True
+
+# By default, BrowserID expects your app to use http://127.0.0.1:8000
+# Uncomment the following line if you prefer to access your app via localhost
+# SITE_URL = 'http://localhost:8000'
+SITE_URL = 'https://' + DOMAIN
+MARKETPLACE_URL = SITE_URL
+MARKETPLACE_OAUTH = {'key': private.MARKETPLACE_OAUTH_KEY,
+                     'secret': private.MARKETPLACE_OAUTH_SECRET}
+BROWSERID_AUDIENCES = [SITE_URL]
+
+STATIC_URL_DOMAIN = 'marketplace-cdn.dev.mozaws.net'
+STATIC_URL = 'https://%s/' % STATIC_URL_DOMAIN
+MEDIA_URL = STATIC_URL + 'mozpay/media/'
+
+# Playdoh ships with Bcrypt+HMAC by default because it's the most secure.
+# To use bcrypt, fill in a secret HMAC key. It cannot be blank.
+HMAC_KEYS = private.HMAC_KEYS
+
+from django_sha2 import get_password_hashers
+PASSWORD_HASHERS = get_password_hashers(base.BASE_PASSWORD_HASHERS, HMAC_KEYS)
+
+# Make this unique, and don't share it with anybody.  It cannot be blank.
+SECRET_KEY = private.SECRET_KEY
+
+# Should robots.txt allow web crawlers?  Set this to True for production
+ENGAGE_ROBOTS = False
+
+# Celery
+BROKER_URL = private.BROKER_URL
+CELERY_ALWAYS_EAGER = False
+CELERY_IGNORE_RESULT = True
+CELERY_DISABLE_RATE_LIMITS = True
+CELERYD_PREFETCH_MULTIPLIER = 1
+
+# Log settings
+
+SYSLOG_TAG = private.SYSLOG_TAG
+# LOGGING = dict(loggers=dict(playdoh = {'level': logging.DEBUG}))
+
+# Uncomment this line if you are running a local development install without
+# HTTPS to disable HTTPS-only cookies.
+SESSION_COOKIE_SECURE = True
+
+ISSUER = DOMAIN
+NOTIFY_ISSUER = DOMAIN
+VERBOSE_LOGGING = True
+
+APP_PURCHASE_KEY = KEY = DOMAIN
```

Comment:
you probably know already; these settings need to match up with zamboni. The comments in the zamboni settings explain: https://github.com/mozilla/zamboni/blob/master/mkt/settings.py#L539